src/IceRpc.Slice/SliceServiceAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) ZeroC, Inc.
+
+namespace IceRpc.Slice;
+
+/// <summary>Represents an attribute used to mark classes implementing Slice services.</summary>
+/// <remarks>The Slice source generator implements <see cref="IDispatcher"/> for classes marked with this attribute.
+/// </remarks>
+[AttributeUsage(AttributeTargets.Class, Inherited = false)]
```

Comment:
Does this mean we can no longer use structs to implement services?

Reply:
Yes, this attribute only applies to classes.

Reply:
I think that's fine:
 - it's not a regression (previously you have to have a class inherit class Service)
 - we could allow struct now but that's the point since you'd immediately box it to use it as a dispatcher